backends/fs.py

```diff
@@ -17,13 +18,32 @@ class FileSystem(StoreBase):
     Save and load data from the local filesystem.
     """
 
-    @staticmethod
-    def _create_dirs(destination):
+    def __init__(self):
+        """Constructor of FileSystem."""
+        self.destination_folder = settings.CUSTOM_DESTINATION_FOLDER
```

Comment:
that is a great PR, just rename CUSTOM_DESTIONATION_FOLDER to ..._PATH 

And where there is the folder word, please rename to path.